samples/captioning/multi-track-captions.html

```diff
@@ -67,6 +70,7 @@
     <body onload="startVideo()">
         <div class="dash-video-player">
             <video controls="true"></video>
+            <div id="ttml-rendering-div"></div>
```

Comment:
@TobbeMobiTV I wonder if we should just copy this sample file and make a specific ebutt named file that adds the div etc.... Not a big deal but just clear to user i think.  Either way works.